tests/experimental/integration/test_migration.py

```diff
@@ -0,0 +1,31 @@
+import py
+import pytest
+
+from great_expectations.data_context.data_context.ephemeral_data_context import (
+    EphemeralDataContext,
+)
+from great_expectations.data_context.data_context.file_data_context import (
+    FileDataContext,
+)
+from tests.test_utils import working_directory
+
+
+@pytest.mark.integration
+def test_convert_to_file_context(
+    tmpdir: py.path.local,
```

Comment:
Can you use `tmp_path` instead of `tmpdir`

https://pytest.org/en/7.2.x/how-to/tmp_path.html#the-tmp-path-fixture

https://pytest.org/en/7.2.x/how-to/tmp_path.html#the-tmpdir-and-tmpdir-factory-fixtures